lib/command.c

```diff
@@ -298,7 +301,9 @@ void install_node(struct cmd_node *node, int (*func)(struct vty *))
 		cmd_token_new(START_TKN, CMD_ATTR_NORMAL, NULL, NULL);
 	graph_new_node(node->cmdgraph, token,
 		       (void (*)(void *)) & cmd_token_del);
-	node->cmd_hash = hash_create(cmd_hash_key, cmd_hash_cmp, NULL);
```

Comment:
`cmd_hash` does nothing and should just be removed entirely

(if we want the duplicate-install warning, we can just put a `bool installed` in `struct cmd_element`)

Reply:
re-implementing cmd_hash a different way is outside the scope of what this change does.  I'd be fine with coming back later and fixing this, but in the meantime let's reduce it's data footprint